README.md

```diff
@@ -20,14 +20,14 @@ Let us know if you want to get involved.
 
 The plugin is designed to support Swift 5 syntax.
 
-| Feature             | Swift       | Objective-C |
-|---------------------|-------------|-------------|
-| Size                | IN PROGRESS | IN PROGRESS |
-| Issues              | IN PROGRESS | IN PROGRESS |
-| Tests               | IN PROGRESS | IN PROGRESS |
-| Coverage            | IN PROGRESS | IN PROGRESS |
-| Complexity          | IN PROGRESS | IN PROGRESS |
-| Syntax highlighting | IN PROGRESS | IN PROGRESS |
+| Feature             | Swift                                                 | Objective-C |
+|---------------------|-------------------------------------------------------|-------------|
+| Size                | IN PROGRESS                                           | IN PROGRESS |
+| Issues              | [SwiftLint](https://github.com/realm/SwiftLint) rules | IN PROGRESS |
```

Comment:
Maybe add a version? New version with new rules will not be taken into consideration but might be reported as issues by users. With a version number they could check if that rule existed and if not, maybe open a PR.

Reply:
Done !